roles/metrics/files/prometheus-node-exporter/node_exporter.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Node Exporter
+
+[Service]
+User=node_exporter
+EnvironmentFile=/etc/default/node_exporter
```

Comment:
I believe there's a node-exporter playbook in the devops repo already. Does this hange the usage, replace it, or otherwise change anything?

Reply:
Ah I wasn't aware of the external role. I just looked through it, they do the same steps just with support for more systems. Do we go with external role with a wider compatibility or with an internal one with a specific setup? 

The playbook would be something that won't need any maintenance or updates, not much should change about it. So I'm fine with either approaches.

Reply:
I'm fine with either approach, let's just avoid duplicates. And if the existing ansible role works for your purposes I'd stick with that.

Reply:
I've removed my node exporter impl. Sticking to the existing role for now.